### DISCUSSION TITLE
Plot customization

### DISCUSSION THREAD

**Discussion by tomasch-cz** [vincentarelbundock/marginaleffects#1173] [Q&A]
Hi, I need help with changing the response curve's colour and width. 
# plot_predictions(gam_model, condition = c('X1'),type = 'response', points=0.05, rug=T)
How can this be done? 
Thank you Tomas

**Comment by vincentarelbundock** [comment]
@tomasch-cz please read this section of the documentation: https://marginaleffects.com/vignettes/plot.html#customization

**Reply by vincentarelbundock** [reply]
FYI, there may be more flexibility in the future. Follow this issue for developments: https://github.com/vincentarelbundock/marginaleffects/issues/1172
